packages/python-sdk/agentstate/langgraph.py

```
@@ -0,0 +1,570 @@
+"""LangGraph checkpoint saver implementations backed by AgentState state storage."""
+
+from __future__ import annotations
+
+import asyncio
+import base64
+import json
+import uuid
+from typing import Any, Dict, Iterator, List, Optional, Sequence, Tuple
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterator`, `Sequence`

Import from `collections.abc`

(UP035)